src/main/java/org/ow2/proactive/catalog/report/TableDataBuilder.java

```diff
@@ -0,0 +1,170 @@
+        createDataHeaderCell(factHeaderrow, (100 / 8f), "Kind");
+        createDataHeaderCell(factHeaderrow, (100 / 8f), "Content Type");
+        createDataHeaderCell(factHeaderrow, (100 / 8f), "Icon");
+        table.addHeaderRow(factHeaderrow);
+    }
+
+    private void createDataHeaderCell(Row<PDPage> factHeaderrow, float width, String title) {
+        Cell<PDPage> cell;
+        cell = factHeaderrow.createCell(width, title);
+        cell.setFontSize(8);
+        cell.setFillColor(java.awt.Color.decode(LIGHT_CYAN));
+        cell.setAlign(HorizontalAlignment.CENTER);
+    }
+
+    private void createDataCell(Row<PDPage> row, float width, String catalogObjectData, int size) {
+        createDataCell(row, width, catalogObjectData, 6, WHITE);
+    }
+
+    private void createDataCell(Row<PDPage> row, float width, String catalogObjectData, int size,
+            String color) {
+        Cell<PDPage> bucketNameCell = row.createCell(width, catalogObjectData);
+        bucketNameCell.setFontSize(size);
+        bucketNameCell.setAlign(HorizontalAlignment.CENTER);
+        bucketNameCell.setFillColor(java.awt.Color.decode(color));
+    }
+
+    private void createDataCell(Row<PDPage> row, float width, String catalogObjectData) {
+        createDataCell(row, width, catalogObjectData, 6);
+    }
+
+    private String getProjectName(CatalogObjectMetadata catalogObject) {
+        return Optional.ofNullable(catalogObject.getProjectName()).orElse("");
+    }
+
+    private String getDescription(CatalogObjectMetadata catalogObject) {
+        return catalogObject.getMetadataList().stream()
+                .filter(metadata -> metadata.getKey().equals("description"))
+                .map(metadata -> metadata.getValue())
+                .map(description -> description.replaceAll("\n", "").replace("\r", "").replace("\t", ""))
+                .findAny().orElse("");
+
+    }
+
+    private String getIcon(CatalogObjectMetadata catalogObject) {
+        return catalogObject.getMetadataList().stream()
+                .filter(metadata -> metadata.getKey().equals("workflow.icon"))
+                .map(metadata -> metadata.getValue()).map(image_url -> {
+                    if (image_url.startsWith("/")) {
+                        return schedulerUrl + image_url;
+                    } else {
+                        return schedulerUrl;
+                    }
+                }).findAny().orElse("");
+
+    }
+
+    private void createEmptyTableRow(BaseTable table) {
+        Row<PDPage> dataRow = table.createRow(10f);
+        createDataCell(dataRow, (100 / 8f), "");
+        createDataCell(dataRow, (100 / 8f), "");
+        createDataCell(dataRow, (100 / 8f), "");
+        createDataCell(dataRow, (100 / 8f) * 2, "");
+        createDataCell(dataRow, (100 / 8f), "");
+        createDataCell(dataRow, (100 / 8f), "");
+        createDataCell(dataRow, (100 / 8f), "");
+    }
+
+    private void createIconCell(Row<PDPage> dataRow, String url_path) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)